input/system/azure/system.go

```diff
@@ -0,0 +1,302 @@
+			logger.PrintError("Azure/System: Failed to make a Cosmos DB client: %v\n", err)
+			return
+		}
+		// Search a server from the list
+		pager := clientFactory.NewClustersClient().NewListPager(nil)
+		for pager.More() {
+			page, err := pager.NextPage(ctx)
+			if err != nil {
+				logger.PrintError("Azure/System: Failed to advance page of Cosmos DB cluster list: %v\n", err)
+				break
+			}
+			for _, v := range page.Value {
+				if v.ID != nil {
+					resourceIDParts := strings.Split(*v.ID, "/")
+					resourceGroup := resourceIDParts[4]
+					resourceType := resourceIDParts[7]
+					resourceName := resourceIDParts[8]
+
+					if config.AzureDbServerName == resourceName {
+						system.Info.Azure = &state.SystemInfoAzure{
+							Location:                 util.StringPtrToString(v.Location),
+							CreatedAt:                util.TimePtrToTime(v.SystemData.CreatedAt),
+							State:                    util.StringCustomTypePtrToString(v.Properties.State),
+							SubscriptionID:           config.AzureSubscriptionID,
+							ResourceGroup:            resourceGroup,
+							ResourceType:             resourceType,
+							ResourceName:             resourceName,
+							CitusVersion:             util.StringPtrToString(v.Properties.CitusVersion),
+							HighAvailabilityEnabled:  util.BoolPtrToBool(v.Properties.EnableHa),
+							CoordinatorStorageMB:     util.Int32PtrToInt(v.Properties.CoordinatorStorageQuotaInMb),
+							NodeStorageMB:            util.Int32PtrToInt(v.Properties.NodeStorageQuotaInMb),
+							CoordinatorVCores:        util.Int32PtrToInt(v.Properties.CoordinatorVCores),
+							NodeVCores:               util.Int32PtrToInt(v.Properties.NodeVCores),
+							CoordinatorServerEdition: util.StringPtrToString(v.Properties.CoordinatorServerEdition),
+							NodeServerEdition:        util.StringPtrToString(v.Properties.NodeServerEdition),
+							NodeCount:                util.Int32PtrToInt(v.Properties.NodeCount),
+						}
+						resourceID = *v.ID
+						break
+					}
+				}
+			}
+		}
+	}
+
+	if resourceID == "" {
+		server.SelfTest.MarkCollectionAspectWarning(state.CollectionAspectSystemStats, "unable to find the database server info")
+		server.SelfTest.HintCollectionAspect(state.CollectionAspectSystemStats, "Make sure the Reader permission of the database is granted to the managed identity.")
+		return
+	}
+
+	// Server metrics
+	client, err := azquery.NewMetricsClient(credential, nil)
+	if err != nil {
+		server.SelfTest.MarkCollectionAspectError(state.CollectionAspectSystemStats, "error making a metrics client: %v", err)
+		logger.PrintError("Azure/System: Failed to make a metrics client: %v\n", err)
+		return
+	}
+
+	// Query metrics data with 1 min interval, for the last 1 min (should return 1 value)
+	metricNames := "cpu_percent,memory_percent,network_bytes_egress,network_bytes_ingress,storage_used"
+	if strings.ToLower(system.Info.Azure.ResourceType) == "flexibleservers" {
```

Comment:
It's "flexibleservers", not "flexibleserver"? Just checking.

Reply:
It is. It was "servers" with a single server.
https://learn.microsoft.com/en-us/azure/templates/microsoft.dbforpostgresql/allversions
And it is "servergroupsv2" with a Cosmos DB.
https://learn.microsoft.com/en-us/azure/templates/microsoft.dbforpostgresql/hsc-allversions